src/js/utils/MesosStateUtil.js

```diff
@@ -2,8 +2,34 @@ import Util from './Util';
 
 const RESOURCE_KEYS = ['cpus', 'disk', 'mem'];
 
+function setSchedulerName(name, tasks) {
+  return tasks.map(function (task) {
```

Comment:
We could change this to the following to make sure that we don't accidentally overwrite a property.

```
return tasks.map(function (task) {
   return  Object.assign({scheduler:name}, task);
}
```
What do you think?

_P.S. We might need to rename the prop to something different e.g. `framework_name` to avoid collisions._